hack/run_in_cloud.sh

```diff
@@ -0,0 +1,48 @@
+#!/bin/bash
+# Note: Tested only with buildah / podman
+
+set -eux
+
+DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+
+if [ -z "${CNCC_REPOSITORY}" ]; then
+	echo "Please set environment variable CNCC_REPOSITORY and point it to your container repository"
+	exit 1
+fi
+
+CONTAINER_ENGINE="${CONTAINER_ENGINE:-podman}"
+BUILDAH_BIN="buildah"
+BUILDAH_BUILD_CMD="buildah bud"
+PODMAN_BIN="podman"
+PODMAN_CMD="podman"
+if [ "$CONTAINER_ENGINE" == "docker" ]; then
+	BUILDAH_BIN="docker"
+	PODMAN_BIN="docker"
+	BUILDAH_BUILD_CMD="docker"
+	PODMAN_CMD="docker"
+fi
+
+for command in oc uuidgen $BUILDAH_BIN $PODMAN_BIN; do
+	if ! command -v $command &> /dev/null
+	then
+		echo "$command not found"
+		exit 1
+	fi
+done
+
+UUID=$(uuidgen)
+IMAGE="${CNCC_REPOSITORY}:${UUID}"
+
+pushd $DIR/../
+	$BUILDAH_BUILD_CMD -t ${IMAGE}
+	$PODMAN_CMD push ${IMAGE}
+popd
+
+oc patch clusterversion version --type json -p '[{"op":"add","path":"/spec/overrides","value":[{"kind":"Deployment","group":"apps","name":"network-operator","namespace":"openshift-network-operator","unmanaged":true}]}]'
+oc scale -n openshift-network-operator deployment.apps/network-operator --replicas=0
+oc patch -n openshift-network-operator deployment.apps network-operator -p '{"spec":{"template":{"spec":{"containers":[{"name":"network-operator","env":[{"name":"CLOUD_NETWORK_CONFIG_CONTROLLER_IMAGE","value":"'${IMAGE}'"}]}]}}}}'
+oc describe  -n openshift-network-operator deployment.apps network-operator | grep CLOUD_NETWORK_CONFIG_CONTROLLER_IMAGE
```

Comment:
tiniest nit of all time. extra space between describe and -n. same happened below on lines 47 and 48 :)

Reply:
It's what happens when one copy/pastes from one's own ~/.bashrc hacks haha thanks!